Savanna/Model/Antelope.cs

```diff
@@ -0,0 +1,127 @@
+﻿namespace Savanna
+{
+    public class Antelope : Animal
+    {
+        /// <summary>
+        /// Antilope entity.
+        /// </summary>
+        public Antelope()
+        {
+            Id = GenerateId();
```

Comment:
Is this Id used anywhere ?

Reply:
Thought that maybe it can be useful in next iterations.
For now Id removed.